hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/transfer/ClassicTransfersTranslator.java

```diff
@@ -64,14 +63,10 @@ public ClassicTransfersTranslator(ClassicTransfersDecoder decoder) {
     @Override
     public boolean matches(@NonNull final HtsCallAttempt attempt) {
         return !attempt.isTokenRedirect()
-                && (Arrays.equals(attempt.selector(), ClassicTransfersTranslator.CRYPTO_TRANSFER.selector())
-                        || Arrays.equals(attempt.selector(), ClassicTransfersTranslator.CRYPTO_TRANSFER_V2.selector())
-                        || Arrays.equals(attempt.selector(), ClassicTransfersTranslator.TRANSFER_TOKENS.selector())
-                        || Arrays.equals(attempt.selector(), ClassicTransfersTranslator.TRANSFER_TOKEN.selector())
-                        || Arrays.equals(attempt.selector(), ClassicTransfersTranslator.TRANSFER_NFTS.selector())
-                        || Arrays.equals(attempt.selector(), ClassicTransfersTranslator.TRANSFER_NFT.selector())
-                        || Arrays.equals(attempt.selector(), ClassicTransfersTranslator.TRANSFER_FROM.selector())
-                        || Arrays.equals(attempt.selector(), ClassicTransfersTranslator.TRANSFER_NFT_FROM.selector()));
+                && (attempt.isSelector(CRYPTO_TRANSFER, CRYPTO_TRANSFER_V2)
```

Comment:
(same comment here - difference is since here the groups are all 2-lists the line wrapping  from our code style doesn't get in the way)